gerrytools/data/census.py

```diff
@@ -201,7 +218,7 @@ def variables(table) -> dict:
     # the descriptor. Each of these tranches should have a width of 6 choose i,
     # where i is the number of categories in the combination. For example, the
     # second tranch (from 13 to 27) has width 15, as 6C2=15.
-    if table in {"P1", "P3"}:
+    if table in {"P1", "P3", "P8", "P10"}:
         tranches = [(3, 8), (11, 25), (27, 46), (48, 62), (64, 69), (71, 71)]
     else:
         tranches = [(5, 10), (13, 27), (29, 48), (50, 64), (66, 71), (73, 73)]
```

Comment:
This is a preexisting issue, but I'm averse to these magic numbers—it seems like this is a opportune place for mistakes to creep if/when we add more tables. I know there's some discussion above of how these tranches are derived, but I think it would be better to make it as explicit as possible, e.g.
```python
tranches = [
    (3, 8),    # column group X
    (11, 25),  # column group Y
    # ...
]
```